packages/ui-kit/src/design-system.js

```diff
@@ -143,6 +143,7 @@ export const typography = {
   lineHeights: {
     small: pxToRem('16px'),
     body: pxToRem('23px'),
+    releaseNoteHeader: pxToRem('28px'),
```

Comment:
I think for sizes we should stick to "normal" names, it's easier then to reuse them.

Now I see this, we should also have the line heights for the markdown headers as variables.

https://github.com/commercetools/commercetools-docs-kit/blob/bee00dd4ee99e66acbfdf8ce9e0298c961e7f6ff/packages/ui-kit/src/components/markdown.js#L8-L64

Let's do that maybe as a follow up.

Reply:
> I think for sizes we should stick to "normal" names, it's easier then to reuse them.

What name would you suggest in this case?

Reply:
Like small, medium, etc. Let's do this as a follow up though.